public/app/plugins/datasource/loki/result_transformer.ts

```diff
@@ -346,8 +346,16 @@ export function lokiStreamsToDataFrames(
   const dataFrame = lokiStreamsToRawDataFrame(data, target.refId);
   enhanceDataFrame(dataFrame, config);
 
-  if (meta.custom && dataFrame.fields.some((f) => f.labels && Object.keys(f.labels).some((l) => l === '__error__'))) {
-    meta.custom.error = 'Error when parsing some of the logs';
+  const labelsField = dataFrame.fields.find((f) => f.name === 'labels');
+
+  if (meta.custom && labelsField) {
+    const allLabelNames = labelsField.values
+      .toArray()
+      .map((labelObject) => Object.keys(labelObject))
+      .flat();
+    if (allLabelNames.some((l) => l === '__error__')) {
+      meta.custom.error = 'Error when parsing some of the logs';
+    }
```

Comment:
this was broken in frontend-mode, because we switched the dataframe-format in the past, but this code did not get updated correctly. as you can see, it's the difference between taking labels from the field's `.labels` prop, and taking data from the labels-field itself.